### PR TITLE
Create Segmentation SDK structure for source and unit tests.

### PR DESCRIPTION
**Pull request by dmandar**

### CODE REVIEW

**Comment by morganchen12** [comment]
Please wait for Ryan/Paul review before merging.

File: FirebaseSegmentation.podspec [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+  
+  s.cocoapods_version = '>= 1.4.0'
+  s.static_framework = true
+  s.prefix_header_file = false
+
+  s.source_files = 'Segmentation/Sources/**/*'
+  s.public_header_files = 'Segmentation/Sources/Public/*.h'
+
+  s.dependency 'FirebaseCore', '~> 6.0'
+  s.dependency 'FirebaseInstanceID', '~> 4.1.1'
```
Comment (paulb777) [inline FirebaseSegmentation.podspec]: Should only depend on minor versions. Change to 4.1

File: Segmentation/Sources/FIRSegmentation.m [hunk 2]
```diff
@@ -0,0 +1,15 @@
+// Copyright 2017 Google
```
Comment (paulb777) [inline Segmentation/Sources/FIRSegmentation.m]: 2019

File: Segmentation/Tests/Unit/Tests-Info.plist [hunk 3]
```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (paulb777) [inline Segmentation/Tests/Unit/Tests-Info.plist]: I'm not sure this file is necessary

File: Segmentation/Tests/Unit/en.lproj/InfoPlist.strings [hunk 4]
```diff
@@ -0,0 +1,2 @@
+/* Localized versions of Info.plist keys */
```
Comment (paulb777) [inline Segmentation/Tests/Unit/en.lproj/InfoPlist.strings]: File isn't necessary. Delete.

**Comment by paulb777** [comment]
If you like, the CI integration can wait until the PR to merge to master. 

To add to CI for `md-floc-master` add the branch name to the bottom of .travis.yml.

File: Segmentation/Tests/Unit/en.lproj/InfoPlist.strings (hunk 4 above)
Comment (dmandar) [reply Segmentation/Tests/Unit/en.lproj/InfoPlist.strings]: Done.

File: FirebaseSegmentation.podspec (hunk 1 above)
Comment (dmandar) [reply FirebaseSegmentation.podspec]: Done.

**Comment by paulb777** [comment]
The diff is unmanageable now. One solution might be to merge master into md-floc-master.

If you haven't yet, review https://github.com/firebase/firebase-ios-sdk/blob/master/AddNewPod.md

**Comment by dmandar** [comment]
> The diff is unmanageable now. One solution might be to merge master into md-floc-master.
> 
> If you haven't yet, review https://github.com/firebase/firebase-ios-sdk/blob/master/AddNewPod.md

I did merge into master. Will do a rebase next time. Mind checking out the individual commit change?
https://github.com/firebase/firebase-ios-sdk/pull/3214/commits/d056de0d65d61717463200db23e424d44a254964

**Pull request merged**